### PR TITLE
fix log layout problem.

### PR DESCRIPTION
**Pull request by quangson91**
Issue: https://github.com/flutter/flutter-intellij/issues/2353

+ Improve logging layout.

Here is the record of how it works.
fix_logging_layout

I'm not good for swing/layout much.
=> please give me a suggestion to make it better.

Notes:
Actually, I want to make log layout still keep visible while user resizes window like this one:
expect_log_view

### CODE REVIEW

File: src/io/flutter/logging/FlutterLogFilterPanel.form [hunk 1]
```diff
@@ -1,25 +1,18 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <form xmlns="http://www.intellij.com/uidesigner/form/" version="1" bind-to-class="io.flutter.logging.FlutterLogFilterPanel">
-  <grid id="27dc6" binding="root" layout-manager="GridLayoutManager" row-count="1" column-count="4" same-size-horizontally="false" same-size-vertically="false" hgap="3" vgap="-1">
+  <grid id="27dc6" binding="root" layout-manager="GridLayoutManager" row-count="1" column-count="3" same-size-horizontally="false" same-size-vertically="false" hgap="-1" vgap="-1">
     <margin top="0" left="0" bottom="0" right="0"/>
     <constraints>
-      <xy x="20" y="20" width="755" height="44"/>
+      <xy x="20" y="34" width="450" height="35"/>
```
Comment (quangson91) [inline src/io/flutter/logging/FlutterLogFilterPanel.form]: I'm not sure is this good.
Actually, I just change the width from `755` to `450`.
So the width of filter action is smaller  -> we can resize without hide filter action.
But if the Intellij IDEA window too narrow -> filter action still hide.
Please give me some suggestion.

File: src/io/flutter/logging/FlutterLogFilterPanel.form [hunk 2]
```diff
@@ -1,25 +1,18 @@
...
     <properties>
+      <background color="-1250068"/>
       <enabled value="true"/>
-      <preferredSize width="986" height="35"/>
+      <minimumSize width="450" height="35"/>
+      <preferredSize width="450" height="35"/>
     </properties>
     <border type="none"/>
     <children>
-      <grid id="9d27c" binding="panelExpression" layout-manager="GridLayoutManager" row-count="1" column-count="1" same-size-horizontally="false" same-size-vertically="false" hgap="-1" vgap="-1">
```
Comment (quangson91) [inline src/io/flutter/logging/FlutterLogFilterPanel.form]: I remove `panelExpression` which is the container of `expressionTextField`.
Instead, I put `expressionTextField` directly in the layout.

File: src/io/flutter/logging/FlutterLogFilterPanel.form [hunk 3]
```diff
@@ -36,11 +29,15 @@
           <text value="Regex"/>
         </properties>
       </component>
-      <hspacer id="6d179">
+      <component id="b7426" class="com.intellij.ui.SearchTextField" binding="textExpression" custom-create="true">
```
Comment (quangson91) [inline src/io/flutter/logging/FlutterLogFilterPanel.form]: Here is the code insert textExpression directly.

File: src/io/flutter/logging/FlutterLogFilterPanel.form (hunk 1 above)
Comment (pq) [reply src/io/flutter/logging/FlutterLogFilterPanel.form]: I'll have to play around.... but this is a vast improvement! 👍

File: src/io/flutter/logging/FlutterLogFilterPanel.form (hunk 3 above)
Comment (pq) [reply src/io/flutter/logging/FlutterLogFilterPanel.form]: 👍

**Pull request merged**